syntax = "proto3";

package grpc;

service PiService {
    rpc SendData (PiRequest) returns (PiReply) {}
} 

message PiRequest{
    string image = 1;
}

message PiReply{
    int32 code = 1;
    string data = 2;
}

// pip install protobuf
// apt install protobuf-compiler
// python -m grpc_tools.protoc -I=. --python_out=. --grpc_python_out=. call_pi.proto